scripts/ci/publish.ts

```diff
@@ -1286,7 +1289,7 @@ function getPatchBranch() {
     const versions = getSemverFromPatchBranch(process.env.BUILDKITE_BRANCH)
     console.debug('versions from patch branch:', versions)
 
-    if (versions?.minor) {
+    if (versions !== undefined) {
```

Comment:
Does that do the same?

Reply:
No, if `versions?.minor` returns `0`, it fails the release. Otherwise, yes because the regex would fail otherwise (it has 2 capturing groups).